modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc

```diff
@@ -93,22 +93,125 @@ image::odata-query-option-attributes.png["odataRequestAttributes is highlighted
 
 == Use The Transform to SQL Operation
 
-APIkit for OData 4 contains a `Transform to SQL Select` operation to transform the input OData `GET` request to a SQL SELECT query. The operation is useful when combined with the `select` operation of the Database Connector. The output payload is a prepared statement and the attributes are a map of prepared statement parameters.
+APIkit for OData 4 contains a `Transform to SQL Select` operation to transform the input OData `GET` request to a SQL SELECT query. The operation is useful when combined with the `select` operation of the Database Connector. The output payload is a prepared statement and the attributes are a map of prepared statement parameters. This operation has been designed to work with the Mule DB connector to help the user expose databases with OData 4 easily.
 
 image::transform-to-sql-transformer.png["Transform to SQL select element and tab are highlighted."]
 
 image::add-db-transform-to-sql.png["Select element and tab are highlighted, as well as the attributes.queryparameters in the input parameters field."]
 
 image::serialize-transform-to-sql.png["Serialized entity element and tab are highlighted, as well as the inbound content in the General section."]
 
+APIkit currently supports the following elements:
+
+* System query options:
+** $filter
+** $select
+** $search
+** $orderBy
+** $skip
+** $top
+* Single entity
+* Entity relation
+* Server-side pagination
+
+APIkit does not currently support the following elements:
+
+* Parameter Aliases
+* Built-in Filter Operations:
+** has
+** divby 
+* Built-in Query Functions, all but:
+** contains
+** startswith
+** endwith
+* $apply
+* $compute
+* $expand
+
+=== How Does Transform to SQL Select Work in Odata 4?
+
+The OData protocol defines a series of conventions that enable clients to query identified resources using URLs. The list of queryable entities, their fields, and the relationships between them are defined in a data model using simple HTTP messages.
+As OData shares some similarities with Java database connectivity (JDBC), the `transform-to-sql-select` operation takes advantage of such similarities to help the user create SQL queries for those resources. The resulting message of the operation is a prepared statement with the SQL `select` sentence as the payload along with the parameters name and values map as the attributes.
+
+=== How Do I Implement the Transform to SQL Select Operation Using Apikit for Odata 4?
+
+<apikit-odata:transform-to-sql-select> is the name of the XML element that represents this operation, which has the following parameters: 
```

Comment:
Please, add back ticks to this XML, as suggested before for a different section

Reply:
Right you are! Thanks!